src/_constants.scss

```diff
@@ -65,9 +65,15 @@ $layout-thumbnail-size: 100px;
 $layout-centered-text-width: 400px;
 $layout-masthead-height: 48px;
 $layout-nav-collapsed-width: 48px;
+$layout-nav-expanded-width: 220px;
 $layout-chart-y-axis-width: $gutter-bigger;
 $layout-chart-x-axis-height: $gutter-big;
 $layout-chart-axis-tick-size: 6px;
 
-$nav-expanded-width: 220px;

```

Comment:
are these not used anymore?

Reply:
no, turns out no one was using it, Navigation.tsx was using its own variable.